test/e2e/framework/kubelet/stats.go

```diff
@@ -578,7 +578,7 @@ func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode)
 		nodeAvgUsage[c].MemoryWorkingSetInBytes /= uint64(len(nodesUsage))
 		nodeAvgUsage[c].MemoryRSSInBytes /= uint64(len(nodesUsage))
 	}
-	result["master"] = masterUsage
+	result["master"] = controlPlaneUsage
```

Comment:
```suggestion
	result["master"] = controlPlaneUsage
```

keeping "master" here?

Reply:
Until we're sure we've cleaned up all the consumers of these metrics it probably makes sense to publish them under both "master" and "controlPlane". This allows easy migration. I agree this will need to be cleaned up though.

Reply:
@SergeyKanzhelev Thanks for the code review. Would you mind if put this in another PR? Since there are already so many changes in this one. And I was not sure of upstream changes required as mentioned by @cheftako Thanks.

Reply:
sure. Please change a description of PR than to not Fix the issue - it will automatically close the issue. Maybe related to issue

> Fixes #94901